src/main.js

```diff
@@ -1,14 +1,29 @@
 // Globals
-window.worldData
 window.game = new Phaser.Game(1000, 600, Phaser.Auto, '');
 
-startGame();
+var Main = function() {};
+var worldData;
```

Comment:
It actually is accessible to a all gamestates the way I have it there, but I agree we should probably do something better with it.